.github/workflows/release.yml

```diff
@@ -77,13 +77,12 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.ref == 'refs/heads/master' }}
     steps:
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@v3
         with:
           name: artifact
           path: dist
 
-      - uses: pypa/gh-action-pypi-publish@master
+      - uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
Do we want to publish from master? I'm publishing MCEq from the releases keep track of what's exactly on pypi.

Reply:
I don't understand the question. It is generally recommended to not user master, but releases instead for all Github Actions.

Reply:
Sorry I confused the + and - :)